validate.py

```diff
@@ -86,6 +87,49 @@ def get_reviews():
 def get_users():
   return list(sorted(os.listdir('players/')))
 
+def get_user_points():
+  points = {}
+
+  for user in get_users():
+    points[user] = 0
+
+    bonus_directory = os.path.join('players', user, 'bonuses')
+    if os.path.isdir(bonus_directory):
+      for named_bonus in os.listdir(bonus_directory):
+        with open(os.path.join(bonus_directory, named_bonus)) as inf:
+          try:
+            points[user] += int(inf.read())
+          except:
+            pass
+    else:
+      # This branch can be deleted after #47 is merged.
+      with open(os.path.join('players', user)) as inf:
+        try:
+          points[user] += int(inf.read())
+        except:
+          pass
+
+  cmd = ['git', 'log', 'master', '--first-parent', '--format=%s']
+  completed_process = subprocess.run(cmd, stdout=subprocess.PIPE)
+  if completed_process.returncode != 0:
+    raise Exception(completed_process)
+  process_output = completed_process.stdout.decode('utf-8')
+
+  merge_regexp = '^Merge pull request #([\\d]*) from ([^/]*)/'
+  for commit_subject in process_output.split('\n'):
+    match = re.match(merge_regexp, commit_subject)
+    if match:
+      pr_number, commit_username = match.groups()
+
+      if int(pr_number) == 33:
```

Comment:
This should be `<= 33`, no?

Reply:
No, if we do `<=` and any pr numbered below 33 merges in the future that will reset points.

Reply:
Just to clarify (IIUC): This is iterating through all commits in reverse chronological order of when they were merged into master stopping when it reaches PR #33. In other words, it is giving a point to everyone who merges a PR after Jeff merges this PR.

This could definitely use some comments :P

Reply:
Please add some comments here to clarify.